src/Symfony/Bundle/DebugBundle/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=5.3.9",
+        "ext-xml": "*",
```

Comment:
Not sure about this. Which part of the DebugBundle would you use without the DI extension?